bin/cli.js

```diff
@@ -176,6 +173,39 @@ const SKIP_CONFIG_VALIDATION = {
   auth: { target: true },
 };
 
+const handleDeprecatedEnvVariables = options => {
+  // HUBSPOT_PORTAL_ID is deprecated, but we'll still support it for now
+  // The HubSpot GH Deploy Action still uses HUBSPOT_PORTAL_ID
+  if (
+    options.useEnv &&
+    process.env.HUBSPOT_PORTAL_ID &&
+    !process.env.HUBSPOT_ACCOUNT_ID
+  ) {
+    uiDeprecatedTag(
+      i18n(`${i18nKey}.injectAccountIdMiddleware.portalEnvVarDeprecated`, {
```

Comment:
nit: should change the i18n property from `injectAccountIdMiddleware` to `handleDeprecatedEnvVariables`